modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1212,6 +1213,47 @@ AUI.add(
 						);
 				},
 
+				syncTranslatedLabelsUI(locale) {
+					const instance = this;
+
+					const defaultLocale = instance.getDefaultLocale();
+
+					if (locale === defaultLocale) {
+						return;
+					}
+
+					const localizationMap = instance.get('localizationMap');
+
+					const labelItem = A.one(
+						'#_com_liferay_dynamic_data_lists_web_portlet_DDLPortlet_PaletteContentBox a[data-value="' +
+							locale +
+							'"] .label'
+					);
+
+					if (labelItem) {
+						if (
```

Comment:
Could we join these two conditions together? Because there is no else in the inner condition.

Reply:
Yes it makes sense. Before I had a different logic and I forgot to change it.